homeassistant/components/cloud/backup.py

```diff
@@ -28,7 +23,7 @@
 from .const import DATA_CLOUD, DOMAIN, EVENT_CLOUD_EVENT
 
 _LOGGER = logging.getLogger(__name__)
-_STORAGE_BACKUP = "backup"
+_STORAGE_BACKUP: Literal["backup"] = "backup"
```

Comment:
Out of curiosity, why do we use a literal here? I'm assuming because of the fact that it can only be one value, but how does a `Final` compare to that?